x-pack/package.json

```diff
@@ -135,6 +136,7 @@
     "polished": "^1.9.2",
     "prop-types": "^15.6.0",
     "puid": "1.0.5",
+    "puppeteer-core": "^1.7.0",
```

Comment:
I think now we can drop `chrome-remote-interface`?

Reply:
Hm. Yes. You're right. I thought I'd done it. This is why we have code-reviews, eh?